simple_history/tests/tests/test_models.py

```diff
@@ -398,8 +398,10 @@ def test_model_with_excluded_fields(self):
         p.save()
         history = PollWithExcludeFields.history.all()[0]
         all_fields_names = [f.name for f in history._meta.fields]
+        most_recent = p.history.most_recent()
         self.assertIn('question', all_fields_names)
         self.assertNotIn('pub_date', all_fields_names)
+        self.assertEqual(most_recent.__class__, PollWithExcludeFields)
 
```

Comment:
Could we also get a test case that ensures that the excluded fields are indeed excluded?

Reply:
@kseever the test here is already checking whether the field is present in the fields of the history model so I think it's already being covered. I have added another statement checking if the field is present in the excluded fields of the history model. It might be that I may have failed to understand your requirement so please feel free to elaborate. =)

Reply:
Ah, I just noticed that the assertion I was looking for already exists on line 403 - confirming that `pub_date` isn't in the list of returned fields. Nevermind me!